src/errorpage.h

```diff
@@ -351,5 +351,6 @@ class TemplateFile
  */
 bool strHdrAcptLangGetItem(const String &hdr, char *lang, int langLen, size_t &pos);
 
+std::ostream & operator <<(std::ostream &os, ErrorState *err);
```

Comment:
Since the type names give us nice info about the parameter we do no need to name them here.

Reply:
I hope that I have understood what you meant here, and will push a patch in a minute. I have also changed other instances of `std::ostream &operator` being defined inconsistently. This is purely a pedantic stylistic change by me: I hope it's OK by you.